views/matrixsplit.html

```diff
@@ -233,8 +233,8 @@
             =
             \begin{bmatrix}
```

Comment:
OK, if you can take a quick look at the results http://github.polettix.it/bezierinfo/#matrixsplit I already did the changes in my fork and I can just submit a new pull request. Thanks!

[I *do have* a raspberry pi by the way... time to take a closer look!]